rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf

```diff
@@ -355,18 +355,22 @@ SecRule REQUEST_HEADERS:Connection "@rx \b(?:keep-alive|close),\s?(?:keep-alive|
 # Check URL encodings
 #
 # -=[ Rule Logic ]=-
-# There are two different chained rules.    We need to separate them as we are inspecting two
-# different variables - REQUEST_URI and REQUEST_BODY.   For REQUEST_BODY, we only want to
-# run the @validateUrlEncoding operator if the content-type is application/x-www-form-urlencoding.
+# There are two different chained rules, 920220 and 920240.
+# We need to separate them as we are inspecting two
+# different variables - REQUEST_URI_RAW and REQUEST_BODY - that
+# need different treatment.
+# For REQUEST_URI_RAW we want to inspect every request in phase 1.
+# For REQUEST_BODY, we only want to inspect if the content-type
+# is application/x-www-form-urlencoding and only in phase 2.
 #
 # -=[ References ]=-
 # http://www.ietf.org/rfc/rfc1738.txt
```

Comment:
Maybe https://www.rfc-editor.org/rfc/rfc2396 is a more updated definition?

Reply:
Yes, with an age of 25 years , the new RFC is probably mature enough.